docs/docusaurus/versioned_docs/version-0.18/cloud/about_gx.md

```diff
@@ -57,34 +51,21 @@ The GX Agent is typically deployed in your organization's deployment environment
 
 GX Cloud deployments can be tailored to meet your specific business requirements. To determine which deployment pattern and GX architectural components are most suitable for your organization, determine how your data is accessed and how users interact with GX Cloud.
```

Comment:
I think this line: 
```
To determine which deployment pattern and GX architectural components are most suitable for your organization, determine how your data is accessed and how users interact with GX Cloud.
```
reads a bit awkwardly.  The doubling of the verb leads to something that feels a little circular: "To determine this, determine that".  Also, I would expect that they would already know how their data is accessed and how they want users to interact with GX Cloud -- I would probably go with:
```
To determine which deployment pattern and GX architectural components are most suitable for your organization, look at how your data is accessed and how users interact with GX Cloud.
```
or:
```
To determine which deployment pattern and GX architectural components are most suitable for your organization, assess how your data is accessed and how users interact with GX Cloud.
```